pubspec.yaml

```diff
@@ -24,4 +21,8 @@ dependencies:
   uuid: ^4.5.1
 
 dev_dependencies:
-  netglade_analysis: ^15.0.0
+  netglade_analysis: ^19.0.0
```

Comment:
rovnou dát 20.0.1 ?

Reply:
To vyžaduje update Dart SDK a to by vyžadovalo update i v aplikacích?